README.md

```diff
@@ -46,19 +53,29 @@ information) can be obtained.  There is no structure imposed at all on the data
 message parts: it is up to the command itself to deserialize however it wishes (e.g. JSON,
 bt-encoded, or any other encoding).
 
+[//]: # Could refer to this as a payload
+
 The Message object also provides methods for replying to the caller.  Simple replies queue a reply
 if the client is still connected.  Replies to service nodes can also be "strong" replies: when
 replying to a SN that has closed connection with a strong reply we will attempt to reestablish a
 connection to deliver the message.  In order for this to work the LokiMQ caller must provide a
 lookup function to retrieve the remote address given a SN x25519 pubkey.
 
+[//]: # Might want different modes of reply here:
+[//]: # - no reply
+[//]: # - opportunistic (reply iff still connected)
+[//]: # - best effort (will open new connection)
```

Comment:
All of these are supported:
- no reply - just don't call `.send_back(...)`/`.send_reply(...)`
- opportunistic - this is the default for non-SN connections, and can be explicitly asked for (i.e. for SNs as well) by adding `send_options::optional{}` into the `send_back/reply(...)` call
- best effort is what you get by default for SNs.  For non SNs it isn't possible.